index.html

```diff
@@ -9,23 +9,48 @@
 </head>
 
 <body>
-  <h1>All awesome books</h1>
+  <nav>
+    <p>Awesome Books</p>
+    <ul>
+      <a href="#" id="showListBtn">List</a>
+      <hr>
+      <a href="#" id="addNewBtn">Add New</a>
+      <hr>
+      <a href="#" id="contactBtn">Contact</a>
+    </ul>
+  </nav>
+  <p id="dateDisplay" class="dateDisplay"></p>
+  <sections id="sections">
+    <section id="listBooks">
+      <h2>All awesome books</h1>
+        <div id="books"></div>
+    </section>
 
-  <div id="listBooks">
-  </div>
-  <hr style="width:50%;text-align:center;margin:20px">
-  
-    <h2>Add a new book</h2>
-    <div class="form">
-    <input id="addTitleInput" type="text" placeholder="title">
-    <input id="addAuthorInput" type="text" placeholder="Author">
-    <button id="addBtn">Add</button>
-    <hr>
-  </div>
-  
+    <section id="addBook">
+      <h2>Add a new book</h2>
```

Comment:
Excellent Job 💯 
![Capture](https://user-images.githubusercontent.com/63972670/205171691-3468e94b-ebe3-41d4-89e2-3b8315e346ac.PNG)

Reply:
Thank you! @OLIPLICHE